tests/console/docker_compose.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test docker-compose installation
+#    Cover the following aspects of docker-compose:
+#      * package can be installed
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+
+use base "consoletest";
+use testapi;
+use utils;
+use strict;
+
+# Setup the required testing environment
+sub setup {
+    # install the docker package if it's not already installed
+    zypper_call("in docker") if script_run("zypper se -x --provides -i docker | grep docker");
+
+    # start the docker daemon if it's not already running
+    systemctl("start docker") if systemctl("status docker");
```

Comment:
Also here, why not just start it regardless? And please use single ticks by default if you don't want perl evaluation in strings by default